src/js/hooks/cycleCount/useCycleCountProductAvailability.jsx

```diff
@@ -0,0 +1,34 @@
+import CycleCountCandidateStatus from 'consts/cycleCountCandidateStatus';
+
+const useCycleCountProductAvailability = (status) => {
+  if (
+    status === CycleCountCandidateStatus.CREATED
+    || status === CycleCountCandidateStatus.REQUESTED
+    || status === CycleCountCandidateStatus.COUNTING
```

Comment:
maybe it'd be a good idea to create some simple util methods to say "isInProgress", "isCounting" etc.